packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -5669,6 +5669,8 @@ Starts a new password recovery transaction for a given user and issues a [recove
 
 > **Note:** A valid `factorType` is required for requests without an API token with administrator privileges. For more information, see [Forgot Password with Trusted Application](#forgot-password-with-trusted-application).
 
+> **Note:** The optional parameter `relayState` can be included as part of the body in the Forgot Password request. `relayState` is a deep link to return to after the response comes back.
```

Comment:
I would not say "deep link" which doesn't seem to mean anything specific.  I would say:

"'relayState' is a link to a site where the user will be redirected when the password recovery operation completes.  The 'relayState' link must point to a trusted origin.

Note: See [Trusted Origins] [#appropriate link]."

You could leave out the Note/link to trusted origins if its too much trouble.

Reply:
Sounds good. I'll make the changes